tensorflow/tools/docker/parameterized_docker_build.sh

```diff
@@ -299,6 +317,26 @@ else
   sed "s/^RUN git clone --branch=.* --depth=1/RUN git clone --branch=${TF_DOCKER_BUILD_DEVEL_BRANCH} --depth=1/" \
       "${ORIG_DOCKERFILE}" > "${DOCKERFILE}"
 
+  # Modify the devel Dockerfile to specify the MKL bazel build parameters
+  if [[ "${TF_DOCKER_BUILD_TYPE}" == "mkl" ]]; then
+    if sed -i -e 's/bazel build -c opt /bazel build -c opt --config=mkl /g' "${DOCKERFILE}"
+    then
+      echo "Modified Dockerfile to add MKL support."
+    else
+      die "FAILED to add MKL support to ${DOCKERFILE}."
+    fi
+  fi
+
+  # Modify the devel Dockerfile with bazel build options
+  if [[ ! -z ${TF_BAZEL_BUILD_OPTIONS} ]]; then
+    if sed -i -e "s/--cxxopt=\"-D_GLIBCXX_USE_CXX11_ABI=0\" /--cxxopt=\"-D_GLIBCXX_USE_CXX11_ABI=0\" ${TF_BAZEL_BUILD_OPTIONS} /g" "${DOCKERFILE}"
```

Comment:
My comment still stands from before. I'd prefer if we could set the necessary Bazel options in a less hacky way.